cmd/query/app/grpc_handler.go

```diff
@@ -73,8 +73,8 @@ func NewGRPCHandler(queryService *querysvc.QueryService,
 		options.Logger = zap.NewNop()
 	}
 
-	if options.Tracer.OT == nil {
-		options.Tracer = jtracer.NoOp()
+	if options.Tracer.OTEL == nil {
+		options.Tracer = *jtracer.NoOp()
```

Comment:
there should be no dereferencing of the pointer, JTracer struct is always passed / stored as a pointer. And the check should be `if options.Tracer == nil {`

Reply:
I will note this hopefully.